playbooks/roles/lets-encrypt/templates/response.yaml.j2

```diff
@@ -0,0 +1,10 @@
+"acme-enter-email": "{{ streisand_admin_email }}"
+"acme-agreement:{{ le_agreements }}": true
+"acmetool-quickstart-choose-server": https://acme-v01.api.letsencrypt.org/directory
```

Comment:
Can you make the value of the `acmetool-quickstart-choose-server` parameter a variable set in `playbooks/roles/lets-encrypt/vars/main.yml`? I find myself switching it to `https://acme-staging.api.letsencrypt.org/directory` before running the playbook for testing purposes and it would be nice to not have to edit the template file to do that.

Reply:
@cpu No problem. It's a role variable now :-)